.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests with node 12
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: CI
+
+on: [push]
+
+jobs:
+  notification_build:
+    name: Notification Module
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: notification
+    steps:
+      - uses: actions/checkout@v2
+      - name: serializing workflow runs
+        uses: softprops/turnstyle@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - run: npm ci
+      - run: npx lockfile-lint --path package-lock.json --allowed-schemes "https:" "git+https:" --allowed-hosts npm github.com
+      - run: npm run test
+        env:
+          ADYEN_SECRET_HMAC_KEY: ${{ secrets.ADYEN_SECRET_HMAC_KEY }}
+          CTP_PROJECT_KEY: adyen-integration-test
+          CTP_CLIENT_ID: ${{ secrets.CTP_CLIENT_ID }}
+          CTP_CLIENT_SECRET: ${{ secrets.CTP_CLIENT_SECRET }}
+
+  extension_build:
+    name: Extension Module
+    needs: notification_build
```

Comment:
I guess we do not need to depend on notification build here. Isn't it?

Reply:
In general yes, here the reason: we are using one commercetools project in both module's tests, so to avoid data conflicts I make them sequential. 
When we introduce a new testing setup we might remove the needs and they could run parallel.

Reply:
ahh ok